src/toil/test/provisioners/clusterTest.py

```diff
@@ -40,7 +41,12 @@ def __init__(self, methodName: str) -> None:
         # We need a boto2 connection to EC2 to check on the cluster.
         # Since we are protected by needs_aws_ec2 we can import from boto.
         import boto.ec2
-        self.boto2_ec2 = boto.ec2.connect_to_region(zone_to_region(self.zone))
+        self.region = zone_to_region(self.zone)
+        self.boto2_ec2 = boto.ec2.connect_to_region(self.region)
+
+        # Get connection to AWS with boto3/boto2
+        self.aws = AWSConnectionManager()
```

Comment:
Does this need to know the default region we took in? Or do we always pass it the region when using it?

Reply:
We should be always passing the region when using it, I don't think we can pass in a default region directly